File: examples/example_boot.rs
use space_net::boot_node::*;

fn main() {
    let boot_server = BootNode::new("network_1", true);
    println!("boot node online..... {:?}", boot_server.get_zid());
    loop {
        boot_server.get_draw_count();
    }
}

File: examples/ExampleBoot.java
import com.example.BootNode;

public class ExampleBoot {
    public static void main(String[] args) {
        BootNode boot_node = new BootNode("network_1",true);
         String zid = boot_node.getZid();
         System.out.println("Boot node zid: " + zid);

         while (true) {
         }
    }
}
